src/components/Header.tsx

```diff
@@ -0,0 +1,22 @@
+export const Header: React.FC = () => {
+  return (
```

Comment:
when you don't have any logic inside component and just return jsx then you can delete the word `return`

and just return jsx in round brackets